packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -56,7 +57,6 @@ import { __DEV__ } from '../env'
 /**
  * Parameters used for Managed Transactions
  */
-const BROWSER_RESPONSIVENESS_INTERVAL = 500
```

Comment:
Why does this needs to be moved back to constants?

Reply:
We use it on transaction-service as well.